src/main/java/org/folio/marccat/dao/persistence/AuthorityNote.java

```diff
@@ -1,11 +1,18 @@
 package org.folio.marccat.dao.persistence;
 
+import org.folio.marccat.dao.SystemNextNumberDAO;
+import org.folio.marccat.shared.CorrelationValues;
+import org.folio.marccat.util.StringText;
+
+import net.sf.hibernate.HibernateException;
+import net.sf.hibernate.Session;
+
 /**
  * @author elena
  *
  */
 public class AuthorityNote extends RecordNote {
-
+  private String noteStringText;
   /**
    * 
```

Comment:
this empty comment block serves no purpose.

Reply:
I have removed it.